client/src/modules/fiscal/fiscal.openingBalance.js

```diff
@@ -26,32 +28,32 @@ function FiscalOpeningBalanceController($state, Accounts, AccountStore, Fiscal,
   vm.gridApi = {};
 
   var columns = [
-    { field : 'number', displayName : '', cellClass : 'text-right', width : 100},
+    { field : 'number', displayName : '', cellClass : 'text-right', width : 100 },
     { field : 'label',
       displayName : 'FORM.LABELS.ACCOUNT',
       cellTemplate : '/modules/accounts/templates/grid.labelCell.tmpl.html',
       headerCellFilter : 'translate',
-      enableFiltering: true,
-      enableColumnMenu: false
+      enableFiltering : true,
+      enableColumnMenu : false,
     },
     { field : 'debit',
       displayName : 'FORM.LABELS.DEBIT',
       headerCellClass : 'text-center',
       headerCellFilter : 'translate',
       cellTemplate : '/modules/fiscal/templates/balance.debit.tmpl.html',
       width : 200,
-      enableFiltering: false,
-      enableColumnMenu: false
+      enableFiltering : false,
+      enableColumnMenu : false,
```

Comment:
Can you use `enableColumnMenus : false` on the `gridOptions`?  It would save a few lines.